ligo-virgo-emfollowup-tutorial.ipynb

```diff
@@ -907,6 +908,64 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## 7. Submitting observation coordinates to GraceDB\n",
+    "\n",
+    "Suppose you have performed some EM observations to follow up on a candidate GW event, and you now want to supply the coordinates of those observations to the LV-EM group.  The first step is to obtain a robotic access password from GraceDB and add it to a netrc file (see Section 2). One can then use the Python GraceDB client in order to submit a list of coordinates to GraceDB. To install the GraceDB client package:\n",
+    "\n",
+    "    $ pip install \"ligo-gracedb>=1.18.dev0\" \n",
+    "\n",
+    "Note: It is highly recommended to install the GraceDB client (as well as other packages described used in this tutorial) inside a [virtual environment](https://virtualenv.pypa.io/en/latest/). This isolates the packages required for interacting with GraceDB from other packages on the system.\n",
+    "Now that the GraceDB client is installed, one can use a script such as this to submit observation records to GraceDB:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from ligo.gracedb.rest import GraceDbBasic, HTTPError\n",
+    "\n",
+    "service = 'https://gracedb.ligo.org/apibasic/'\n",
+    "g = GraceDbBasic(service)\n",
+    "\n",
+    "graceid = 'T125706'\n",
+    "raList = '45.0,47.0,49.0'\n",
+    "raWidthList = '2.0'\n",
+    "decList = '45.0,47.0,49.0'\n",
+    "decWidthList = '2.0'\n",
+    "startTimeList = '2015-05-01T12:30:10.95,2015-05-01T12:31:10.95,2015-05-01T12:32:10.95'\n",
+    "durationList = '100.0,100.0,100.0'\n\n",
+
```

Comment:
I did a bit of experimentation with sending lists in the request body, but I never converged on it. Probably the best way of handling this would be in the client (rather than the server). I'll make that a feature request.